docs/cloud/satellites/best-practices.md

```diff
@@ -0,0 +1,90 @@
+
+Many CI systems allow this behavior to be configured. For example, on Github Actions the concurrency block can be used:
+
+```yaml
+name: Github Actions CI
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  ... 
+```
+
+## Determining when Satellites are Overloaded
+
+A satellite may become sluggish, or even fail a build (in extreme cases) if it has taken on more jobs than it has the resources to handle. If you are experiencing a drop in performance, consider using a larger instance, or splitting up your build across multiple satellites. Note that in some cases, performance issues can be the result of a problem in the build script or external factors such as network operations.
+
+There are some metrics printed by the Earthly CLI that can be used to gauge the performance and overall health of a satellite. These metrics are printed at the start of the build during the “Init” phase, when running the satellite inspect command, or sometimes printed midway through the build if Earthly detects a performance problem. Here is an example of the metrics printed during Init:
+
+```
+ Init 🚀
+————————————————————————————————————————————————————————————————————————————————
+
+           satellite | Connecting to my-satellite...
+           satellite | ...Done
+           satellite | Version github.com/earthly/buildkit v0.7.23 086f60eecf5a2261bbd53299614f88b3def12746
+           satellite | Platforms: linux/amd64 (native) linux/amd64/v2 linux/amd64/v3 linux/amd64/v4 linux/arm64 linux/riscv64 linux/ppc64 linux/ppc64le linux/s390x linux/386 linux/mips64le linux/mips64 linux/arm/v7 linux/arm/v6
+           satellite | Utilization: 0 other builds, 0/12 op load
+           satellite | GC stats: 2.1 MB cache, avg GC duration 0s, all-time GC duration 23ms, last GC duration 0s, last cleared 0 B
+```
+
+In this output, there are a couple of metrics which are worth keeping an eye on:
+* `Utilization` – The number of other concurrent builds and op load usage. The op load indicates the number of concurrent operations being executed, compared to the max allowed by the satellite, where an operation is a step within an Earthfile. When the op load is at the max, operations may begin to queue, resulting in slower performance.
+* `GC Stats` – The amount of cache on disk, and time spent doing garbage collection. Compare this to the amount of [disk available per the satellite’s size](https://earthly.dev/pricing). The cache typically hovers around 50% of the total disk size, however, if the cache size is close to the max, then aggressive garbage collection can cause degraded performance.
+
+On top of the metrics mentioned above, these other symptoms may indicate an overloaded satellite:
+* “No space left on device” errors
```

Comment:
stick with the ascii double-quote
```suggestion
* "No space left on device" errors
```

Reply:
This slipped through a copy-and-paste from google doc

Reply:
figured as much -- nice to have google docs take care of spell checking though :)